client/boot/index.js

```diff
@@ -295,7 +295,7 @@ function reduxStoreReady( reduxStore ) {
 		next();
 	} );
 
-	if ( config.isEnabled( 'oauth' ) ) {
+	if ( config.isEnabled( 'oauth' ) && config.isEnabled( 'desktop' ) ) {
```

Comment:
In the web app the login screen makes a request to the node server. This will only set cookies in the local environment not on WordPress.com, so users will have to log in again to go to wp-admin.